packages/mediapipe-task-text/build.dart

```diff
@@ -0,0 +1,97 @@
+import 'dart:io';
+import 'package:native_assets_cli/native_assets_cli.dart';
+import 'package:http/http.dart' as http;
+import 'package:path/path.dart' as path;
+
+import 'sdk_downloads.dart';
+
+late File logFile;
+
+final logs = <(DateTime, String)>[];
+void log(String msg) {
+  logs.add((DateTime.now(), msg));
+  if (!logFile.parent.existsSync()) {
+    logFile.parent.createSync();
+  }
+
+  if (logFile.existsSync()) {
+    logFile.deleteSync();
+  }
+  logFile.createSync();
+  logFile.writeAsStringSync(logs
+      .map<String>((rec) => '[${rec.$1.toIso8601String()}] ${rec.$2}')
+      .toList()
+      .join('\n\n'));
+}
+
+Future<void> main(List<String> args) async {
+  logFile = File(
+    path.joinAll([
+      Directory.current.path, // root dir of app using `mediapipe-task-xyz`
+      'build/${DateTime.now().millisecondsSinceEpoch}-build-log.txt',
```

Comment:
In standalone dart, there is no such thing as a build dir.
Also, in Flutter, the build dir is configurable.

Consider writing the logs to `buildConfig.outDir.resolve('build-log.txt')`.
And consider appending new logs instead of creating a file for every millisecond. (Also you're overwritting multiple logs in the same millisecond currently.)

Also, consider writing out the logs to stdout in addition to a file.

Reply:
> In standalone dart, there is no such thing as a build dir. Also, in Flutter, the build dir is configurable.

The `mediapipe-task-text` package can't be used in standalone Dart - though maybe it should be able to 🤔 . Either way, the second detail is potentially trouble! I'll move these logs to the location you suggested.

> And consider appending new logs instead of creating a file for every millisecond. (Also you're overwritting multiple logs in the same millisecond currently.)

Admittedly, this part is a little confusing. I write logs to the file every time because, during development, this whole process was crashing on me for opaque reasons and I wanted to capture as much information as possible. Regarding overwriting - that value is captured right at the start of the whole process and does not change, so I don't believe any logs will collide (unless they truly start in the same millisecond).

> Also, consider writing out the logs to stdout in addition to a file.

I got into this crazy business because this process's stdout is not exposed to the developer (flutter/flutter#136386).

Also, all of this logging noise is not really code I ultimately want to keep in here. But, I've removed it twice so far and both times later had to add it back - so for now I just left it in to support further development leading up to our initial release.